.rubocop.yml

```diff
@@ -0,0 +1,13 @@
+
+Style/Documentation:
+  Enabled: false
+
+Metrics/MethodLength:
+  Enabled: false
+
+Metrics/AbcSize:
+  Enabled: false
+
+Metrics/ParameterLists:
+  Enabled: false
```

Comment:
You disabled these rules for the whole project.
Please, use `exclude:` option for the files to be ignored by `rubocop`.